### PR TITLE
Add React Hooks

### PR DESCRIPTION
**Pull request by glauberfc**
**[Awesome Hooks](https://github.com/glauberfc/awesome-hooks)**

**A list about React Hooks new feature with very good links about this subject**

PR reviews:
#1482
#1451

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

### CODE REVIEW

**Comment by angoca** [comment]
The list does not includes a succinct description: **Includes a succinct description of the project/theme at the top of the readme.**

**Comment by OvermindDL1** [comment]
As an aside calling is `Awesome-Hooks` is not descriptive of what kind of hooks, like I'd initially think it would be a listing of awesome hooks like awesome githooks or awesome webhooks.  I'd never think it would be for react.  ^.^;

**Comment by glauberfc** [comment]
Thanks, @OvermindDL1. I updated the list's title now for Awesome React Hooks

**Comment by glauberfc** [comment]
> The list does not includes a succinct description: Includes a succinct description of the project/theme at the top of the readme.

@angoca as you can see on others awesome lists like on [Awesome Relay](https://github.com/expede/awesome-relay#readme) the description on the list itself is about the content of the list, but the description on [Awesome Lists](https://github.com/sindresorhus/awesome) need to follows this rule that you pointed. So, I think that my list is correct

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)

---

> Includes a project logo/illustration whenever possible.
**The image should be high-DPI. Set it to maximum half the width of the original image.**

---

> The description starts with an uppercase character and ends with a period.

---

`Extensions/Libs` => `Extensions/Libraries`

---

https://github.com/glauberfc/awesome-react-hooks/blob/master/contributing.md is just copy-pasted from Awesome. It's expected that you have one customized for your list. Like: https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/glauberfc/awesome-react-hooks
✖ Linting

  README.md:48:3
  ✖   8:1    ToC item "Hooks (proposal)" link "https://github.com/glauberfc/awesome-hooks#hooks-proposal" not found  awesome/toc
  ✖  25:96   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  26:106  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  27:93   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  28:85   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  29:61   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  30:87   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  31:80   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  32:81   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  33:73   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  34:76   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  35:57   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  36:136  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  37:112  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  38:100  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  39:99   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  40:117  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  41:118  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  42:113  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  43:127  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  44:171  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  47:106  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  48:3    Link to https://github.com/reactjs/rfcs/blob/hooks-rfc/text/0000-react-hooks.md is dead                 no-dead-urls
  ✖  49:66   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  50:110  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  57:51   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  59:87   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  60:49   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  61:36   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  65:85   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  68:76   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  69:54   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  70:59   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  71:87   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  72:71   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  73:56   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  74:56   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  75:78   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  76:103  List item description must end with proper punctuation                                                  awesome/list-item
  ✖  77:85   List item description must end with proper punctuation                                                  awesome/list-item
  ✖  78:66   List item description must end with proper punctuation                                                  awesome/list-item

  41 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
> as you can see on others awesome lists like on [Awesome Relay](https://github.com/expede/awesome-relay#readme) the description on the list itself is about the content of the list, but the description on [Awesome Lists](https://github.com/sindresorhus/awesome) need to follows this rule that you pointed. So, I think that my list is correct

The guidelines are correct. Don't follow existing lists. They might not all be up to date with the latest guidelines.

**Comment by sindresorhus** [comment]
Ping

File: readme.md [hunk 1]
```diff
@@ -191,6 +191,7 @@
 	- [Protips](https://github.com/AllThingsSmitty/css-protips#readme)
 - [React](https://github.com/enaqx/awesome-react#readme) - App framework.
 	- [Relay](https://github.com/expede/awesome-relay#readme) - Framework for building data-driven React apps.
+	- [React Hooks](https://github.com/glauberfc/awesome-react-hooks#readme) - A new feature proposal that lets you use state and other React features without writing a class.
```
Comment (sindresorhus) [inline readme.md]: It's no longer just a proposal.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**